content/en/docs/v3.5/op-guide/etcd3_alert.rules.yml

```diff
@@ -0,0 +1,148 @@
+      summary: etcd grpc requests are slow
+    expr: |
+      histogram_quantile(0.99, sum(rate(grpc_server_handling_seconds_bucket{job=~".*etcd.*", grpc_method!="Defragment", grpc_type="unary"}[5m])) without(grpc_type))
+      > 0.15
+    for: 10m
+    labels:
+      severity: critical
+  - alert: etcdMemberCommunicationSlow
+    annotations:
+      description: 'etcd cluster "{{ $labels.job }}": member communication with {{ $labels.To }} is taking {{ $value }}s on etcd instance {{ $labels.instance }}.'
+      summary: etcd cluster member communication is slow.
+    expr: |
+      histogram_quantile(0.99, rate(etcd_network_peer_round_trip_time_seconds_bucket{job=~".*etcd.*"}[5m]))
+      > 0.15
+    for: 10m
+    labels:
+      severity: warning
+  - alert: etcdHighNumberOfFailedProposals
+    annotations:
+      description: 'etcd cluster "{{ $labels.job }}": {{ $value }} proposal failures within the last 30 minutes on etcd instance {{ $labels.instance }}.'
+      summary: etcd cluster has high number of proposal failures.
+    expr: |
+      rate(etcd_server_proposals_failed_total{job=~".*etcd.*"}[15m]) > 5
+    for: 15m
+    labels:
+      severity: warning
+  - alert: etcdHighFsyncDurations
+    annotations:
+      description: 'etcd cluster "{{ $labels.job }}": 99th percentile fsync durations are {{ $value }}s on etcd instance {{ $labels.instance }}.'
+      summary: etcd cluster 99th percentile fsync durations are too high.
+    expr: |
+      histogram_quantile(0.99, rate(etcd_disk_wal_fsync_duration_seconds_bucket{job=~".*etcd.*"}[5m]))
+      > 0.5
+    for: 10m
+    labels:
+      severity: warning
+  - alert: etcdHighFsyncDurations
+    annotations:
+      description: 'etcd cluster "{{ $labels.job }}": 99th percentile fsync durations are {{ $value }}s on etcd instance {{ $labels.instance }}.'
+      summary: etcd cluster 99th percentile fsync durations are too high.
+    expr: |
+      histogram_quantile(0.99, rate(etcd_disk_wal_fsync_duration_seconds_bucket{job=~".*etcd.*"}[5m]))
+      > 1
+    for: 10m
+    labels:
+      severity: critical
+  - alert: etcdHighCommitDurations
+    annotations:
+      description: 'etcd cluster "{{ $labels.job }}": 99th percentile commit durations {{ $value }}s on etcd instance {{ $labels.instance }}.'
+      summary: etcd cluster 99th percentile commit durations are too high.
+    expr: |
+      histogram_quantile(0.99, rate(etcd_disk_backend_commit_duration_seconds_bucket{job=~".*etcd.*"}[5m]))
+      > 0.25
+    for: 10m
+    labels:
+      severity: warning
+  - alert: etcdBackendQuotaLowSpace
+    annotations:
+      description: 'etcd cluster "{{ $labels.job }}": database size exceeds the defined quota on etcd instance {{ $labels.instance }}, please defrag or increase the quota as the writes to etcd will be disabled when it is full.'
+      summary: etcd cluster database is running full.
+    expr: |
+      (etcd_mvcc_db_total_size_in_bytes/etcd_server_quota_backend_bytes)*100 > 95
```

Comment:
this has been changed at https://github.com/etcd-io/etcd/pull/14237

Reply:
```suggestion
(last_over_time(etcd_mvcc_db_total_size_in_bytes[5m]) / last_over_time(etcd_server_quota_backend_bytes[5m]))*100 > 95
```

Reply:
As it says in the file, it was generated from the v3.5 mixin,  https://github.com/etcd-io/etcd/blob/release-3.5/contrib/mixin/mixin.libsonnet